### PR TITLE
Apply sentence case formatting to Setting Sidebar controls

### LINKED ISSUE #19902: Copy: Setting Sidebar controls case inconsistencies
**Describe the bug**
In support of utilizing sentence case (as we've started implementing in #16764, #18758 and #19900), block controls within a block's `InspectorControls` should have sentence case formatting.

Related/expands on: #16764, #12366, #19900

**Screenshots**
Here are a few, though there are a number of control labels that need adjusting:

Screen Shot 2020-01-26 at 4 29 43 PM
Screen Shot 2020-01-26 at 4 30 24 PM

### PR DESCRIPTION
**Pull request by richtabor**
## Description
This PR closes #19902 by changing existing PanelBody titles to utilize sentence case formatting. Related to #18758, #16764, and #19902.

## Screenshots 

Screen Shot 2020-01-26 at 5 20 06 PM

Screen Shot 2020-01-26 at 5 20 17 PM

## Types of changes
Text string changes.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR. .

### CODE REVIEW

File: storybook/test/__snapshots__/index.js.snap [hunk 1]
```diff
@@ -8,7 +8,7 @@ exports[`Storyshots Components/Animate Appear Bottom Left 1`] = `
     className="components-notice__content"
   >
     <p>
-      Appear animation. Origin:
```
Comment (jasmussen) [inline storybook/test/__snapshots__/index.js.snap]: The line breaks in this paragraph are a little weird, with a period on its own line. But will the removed space not mean it says "Origin:bottom left" now? I haven't yet had time to dive deep into storybook stuff, so I could be wrong.

**Comment by jasmussen** [comment]
Aside from a possibly-storybook specific question (which applies to a few subsequent changes also) this is a really lovely consistency PR, thank you.

CC: @karmatosed — I believe you touched a menu recently, and I think you were moving in the same direction? But here, have a ping regardless.

**Comment by karmatosed** [comment]
I am super keen on anything that brings sentence case more consistently throughout, thanks for this @richtabor

File: storybook/test/__snapshots__/index.js.snap (hunk 1 above)
Comment (ItsJonQ) [reply storybook/test/__snapshots__/index.js.snap]: These are automatically generated by Storybook's test snapshots. We can typically ignore these updates during review.

Hope this helps!

**Review by ItsJonQ — APPROVED** [review]
🚀 from me! Code changes look good + consistent. Great job @richtabor !

**Comment by youknowriad** [comment]
Anything blocking merge here? can we rebase and land?

**Comment by richtabor** [comment]
> Anything blocking merge here? can we rebase and land?

I don't think there is anything blocking - just slipped by.

**Comment by mkevins** [comment]
Hi @richtabor :wave: ,

Thanks for these string changes. I wonder if it may be worth also including similar changes in `*.native.js` files in this PR for consistency. For example: https://github.com/WordPress/gutenberg/blob/9bec6eae979ff2546d29589bc31097974140e4ac/packages/block-library/src/image/edit.native.js#L420

@iamthomasbishop  :wave: , do we want this same change for mobile? We share some of these strings on Gallery block, and I believe we have made some [similar changes](https://github.com/WordPress/gutenberg/pull/19901/commits/969a29484a177db06279b6d2dd61a78e632a5e31#diff-a61b3f346c11f42236eb2b1a783bf9bfR334) recently to some strings in `*.native.js` files:  Wdyt?

**Comment by iamthomasbishop** [comment]
> do we want this same change for mobile?

Great (and timely) question. This came up recently as part of another discussion re: native typography, and here's where I stand on the topic of casing wrt platform standards. My opinion is that we should default to _Sentence case_ (which aligns well with Android), but if possible allow iOS to force _Title Case_ where necessary (titles, buttons, etc).

I imagine this would be pretty straight-forward to do with just CSS, but I'm not sure if in our RN project it is that straight-forward, so I'd defer to your best judgment @mkevins. We'd want to apply it programmatically, in a systemized way, so it can scale to the whole UI and be relatively future-proof. An added benefit: y'all devs wouldn't have to even worry about which case to write in 😄

**Comment by jorgefilipecosta** [comment]
I rebased and merged this PR so it is included in the next WordPress beta release. I think as follow up we should update the strings on native files.

**Pull request merged**